ck.sh

```diff
@@ -113,11 +92,16 @@ case "$COMMAND" in
 				# Replace with new build files.
 				cp -r dev/builder/release/ckeditor/* ../ckeditor/
 
-				str=$(printf "%s\n" "Don't forget to commit the result!\n\n" \
-					"    git add -A -- ckeditor\n" \
-					"    git commit -m 'Update CKEDITOR'")
-
-				message "✅  DONE" "$str"
+				echo
+				echo "*---------------------------------------------------------*"
+				echo "|                        ✅ DONE                          |"
+				echo "*---------------------------------------------------------*"
+				echo
+				echo "Don't forget to commit the result!"
+				echo
+				echo "git add -A -- ckeditor"
+				echo "git commit -m 'Update CKEDITOR'"
```

Comment:
Just a minor nit; what would be an example of a good commit message here? If we can show one automatically, great; otherwise we should probably leave it off (your instruction to "commit the result" is probably enough). I just don't won't the history to wind up with a bunch of messages that all say "Update CKEDITOR", because you'll have to look at the commits (which may be large) in order to figure out what that actually means.